lib/router.js

```diff
@@ -182,8 +180,7 @@ Router.prototype.render = function render(routeName, params, query) {
 
     var _path = route.spec.path;
     var _url = _path.replace(/\/:([A-Za-z0-9_]+)(\([^\\]+?\))?/g, pathItem);
-    var items = Object.keys(query || {}).map(queryItem);
-    var queryString = items.length > 0 ? ('?' + items.join('&')) : '';
+    var queryString = Object.keys(query || {}).length ? ('?' + qs.stringify(query)) : '';
```

Comment:
This line is too long, and is failing the tests currently. Please reduce it to less than 80 characters.

It might be helpful to move `Object.keys.length` call to a separate line. Similar to how it was before, with items.